src/app/components/MediaLoader/utils/buildSettings.test.ts

```diff
@@ -1111,5 +1121,100 @@ describe('buildSettings', () => {
         showAds: false,
       });
     });
+
+    it('Should process a live media block with service ID into a valid playlist item', () => {
+      const mediaBlock = {
+        type: 'liveMedia',
+        model: {
+          urn: 'urn:bbc:pips:sid:bbc_arabic_tv',
+          title: 'BBC Arabic TV',
+          type: 'episode',
+          synopses: {
+            short:
+              'يلقي برنامج إكسترا التلفزيوني الضوء على قصص إنسانية فريدة من حياتنا، نعيشها ونرويها للشباب العربي',
+            medium:
+              'يلقي برنامج إكسترا التلفزيوني الضوء على قصص إنسانية فريدة من حياتنا، نعيشها ونرويها للشباب العربي',
+            long: 'يلقي برنامج إكسترا التلفزيوني الضوء على قصص إنسانية فريدة من حياتنا، نعيشها ونرويها للشباب العربي',
+          },
+          mediaType: 'audio_video',
+          imageUrlTemplate:
+            'https://ichef.bbci.co.uk/images/ic/$recipe/p0hwr92l.jpg',
+          masterbrand: {
+            id: 'bbc_arabic_tv',
+            name: 'تلفزيون بي بي سي عربي',
+            networkName: 'تلفزيون بي بي سي عربي',
+            type: 'tv',
+            imageUrlTemplate: 'ichef.bbci.co.uk/images/ic/$recipe/p08b23t4.png',
+          },
+          version: {
+            vpid: 'n4pdlz1s10',
+            duration: 'PT30M',
+            availabilityType: 'simulcast',
+            versionTypes: [{ type: 'Original', name: 'Original version' }],
+            schedule: null,
+            serviceId: 'bbc_arabic_tv',
+            authToken: null,
+            status: 'LIVE',
+            warnings: null,
+          },
+          leadMedia: true,
+        },
+      };
+
+      const result = buildSettings({
+        ...arabicMediaBaseSettings,
+        blocks: [mediaBlock] as MediaBlock[],
+        pageType: LIVE_PAGE,
+      });
+
+      expect(result).toStrictEqual({
+        mediaType: 'video',
+        playerConfig: {
+          appName: 'news-arabic',
+          appType: 'responsive',
+          autoplay: false,
+          counterName: 'live_coverage.cvp5r6m6mgpt.page',
+          enableToucan: true,
+          playlistObject: {
+            holdingImageURL:
+              'https://ichef.bbci.co.uk/images/ic/$recipe/p0hwr92l.jpg',
+            items: [
+              {
+                duration: 1800,
```

Comment:
Does it make sense to have a duration for a live stream here? Should it be omitted?